src/agenteval/cli.py

```diff
@@ -598,6 +605,32 @@ def publish_lb_command(repo_id: str, submission_urls: tuple[str, ...]):
             ) as f:
                 f.write(lb_submission.model_dump_json(indent=None))
 
+        # Validate the config with the schema in HF
+        readme = Readme.download_and_parse(repo_id)
+        missing_configs = set(config_splits.keys()) - set(readme.configs.keys())
+        if missing_configs:
+            click.echo(
+                f"Config name {list(missing_configs)} not present in hf://{repo_id}/README.md"
+            )
+            click.echo("Run 'update_readme.py add-config' to add it")
```

Comment:
🎉

Reply:
This logic knows what results repo id is being used, but I think the update_readme.py will default to asta-bench-internal-results if run as stated here. Could the suggested command here include repo id for clarity?